### ISSUE TITLE
Warning build in action. Not detecting the scripts.

### ISSUE THREAD

**Issue by hellresistor** [luizm/action-sh-checker#12]
I am getting this Warning during action build. (use the v.0.1.5)

https://github.com/hellresistor/BCNA-Installer/runs/643824056?check_suite_focus=true
Line: 80

```
WARNING: Ignoring APKINDEX.70f61090.tar.gz: No such file or directory 
WARNING: Ignoring APKINDEX.ca2fea5b.tar.gz: No such file or directory
```

I am trying apply your action on one of my shellscripts but are saying "No shell scripts found"
.yml file: https://github.com/hellresistor/BCNA-Installer/actions/runs/95515052/workflow

Maybe issue because my script filename contains a " - " ?!

Are you working on V0.1.5 , right? are happening with v0.1.4 too (tooo slow) :|

**Comment by luizm** [comment]
> I am trying apply your action on one of my shellscripts but are saying "No shell scripts found"

Did you do a checkout before running this action?  

Example:

```
    steps:
      - uses: actions/checkout@v1
      - name: Run the sh-checker
        uses: luizm/action-sh-checker@v0.1.4
        env:
            .............
            .............
````

**Comment by hellresistor** [comment]
Oh.. my ... freaked head... I am so wasteedd!! YES I FORGOT HAHAHA Jizzz

**Comment by luizm** [comment]
> Oh.. my ... freaked head... I am so wasteedd!! YES I FORGOT HAHAHA Jizzz

No problem :)
